Update university-of-miami.mdx

glichtenstein opened the pull request:
we are changing the node to the Sylvester Cancer Center, it is part of University of Miami but Sylvester is our main place of work at the UM's Miller School of Medicine. We will soon comfirm the venue, we are trying to book a nice room with good wifi, as our meeting room may be not ideal. thanks!

@netlify /events/2025/hackathon-march-2025/university-of-miami

mashehu reviewed (changes requested):
Please change the entry in locations accordingly

glichtenstein commented:
> Please change the entry in locations accordingly

@mashehu Thanks for the review! I added the new denomination in the index as well (Sylvester Comprehensive Cancer Center) and added two more contacts as hosts. Please let me know if it works, and if any additional changes are required. P.S: We are working to comfirm the venue ASAP 🤞

ewels commented on sites/main-site/src/content/events/2025/hackathon-march-2025/university-of-miami.mdx:
```diff
@@ -19,4 +19,10 @@ Attendees from outside the University of Miami **can** attend.
 
 Main contact: Franklin Sotolongo ([fsotolongo@med.miami.edu](mailto:fsotolongo@med.miami.edu))
 
+Hosts:
+Vasileios Stathias, PhD - Assistant Director, Data Science - ([v.stathias@med.miami.edu](mailto:v.stathias@med.miami.edu))
+Gabriel Lichtenstein, PhD - Senior Bioinformatics Analyst - ([gxl674@med.miami.edu](mailto:gxl674@med.miami.edu))
+Sylvester Comprehensive Cancer Center
+Molecular and Cellular Pharmacology, University of Miami
```
```suggestion
Hosts:

- Vasileios Stathias, PhD - Assistant Director, Data Science - ([v.stathias@med.miami.edu](mailto:v.stathias@med.miami.edu))
- Gabriel Lichtenstein, PhD - Senior Bioinformatics Analyst - ([gxl674@med.miami.edu](mailto:gxl674@med.miami.edu))

Sylvester Comprehensive Cancer Center
Molecular and Cellular Pharmacology, University of Miami
```

ewels commented:
@nf-core-bot fix linting

ewels reviewed (approved):
Thanks!
